tests/unit/testplan/testing/multitest/entries/test_assertions.py

```diff
@@ -817,15 +817,15 @@ def test_evaluate_false(
     ("abc\r\nxy z\r\n", "abc \nxy\t\tz\n", True, False, False, True, False),
     (" abc\r\nxyz\r\n", "abc \nx y\tz\n", False, True, False, False, True),
     ("abc\n\nxyz\n", "abc\nxyz\n\n", False, False, True, False, False),
-    ("abc\nxyz\n", "abc\r\n\r\nxyz", False, True, True, True, True),
+    ("abc\nxyz\n", "abc\r\n\r\nxyz", False, True, True, True, False),
```

Comment:
Unified and context are display options. https://www.gnu.org/software/diffutils/manual/html_node/Context-Format.html and https://www.gnu.org/software/diffutils/manual/html_node/Unified-Format.html Having them both true is no problem to difflib so I expect it to work as before. Having it both true for the diff command is an issue as it makes for a failing diff command due to the conflict. Since for Windows it is still difflib and pure Python, I think nothing will change for internal.

Reply:
I see!